MetroscopeModelingLibrary/Tests/Multifluid/HeatExchangers/Fogging_reverse.mo

```diff
@@ -0,0 +1,96 @@
+within MetroscopeModelingLibrary.Tests.Multifluid.HeatExchangers;
+model Fogging_reverse
+
+  // Boundary conditions
+  input Real P_fg_in(start = 1.1, min = 0, nominal = 10) "barA";
+  input Utilities.Units.MassFlowRate Q_fg_in(start = 660) "kg/s";
+  input Real T_fg_in(start = 28.8)  "degC";
+
+  input Real P_water(start = 5, min = 0, nominal = 10) "barA";
+  input Real Q_water(start = 2.5) "kg/s";
+  input Real T_water(start = 19.86, min = 0, nominal = 50) "degC";
+
+  // Calibrated parameters
+  input Utilities.Units.MassFraction x_vapor(start=1);
+
+  // Calibration inputs
+  output Real T_fg_out(start=23.5) "degC";
```

Comment:
Here the comments are not very clear, do we calibrate something or not ? If yes, we should have a clear reverse model and a direct model

Reply:
Well in fact it is still not clear for me here, @LucileD-met do we calibrate `x_vapor` ?